bitrise.yml

```diff
@@ -2,173 +2,181 @@ format_version: 3
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -ex
 
-            # Check for unhandled errors
-            go get -u github.com/kisielk/errcheck
+                # Check for unhandled errors
+                go get -u github.com/kisielk/errcheck
 
-            # Go lint
-            go get -u golang.org/x/lint/golint
+                # Go lint
+                go get -u golang.org/x/lint/golint
   create-release:
     steps:
-    - ensure-clean-git: {}
-    - script:
-        title: bump version number, then merge into 'prod' branch, tag & commit &
-          push
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -e
-
-            echo
-            echo 'How to roll-back?'
-            echo '* if you want to undo the last commit you can call:'
-            echo '    $ git reset --hard HEAD~1'
-            echo '* to roll back to the remote state:'
-            echo '    $ git reset --hard origin/[branch-name]'
-            echo
+      - script:
+          title: Create release commints
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -e
+                
+                echo
+                echo 'How to roll-back?'
+                echo '* if you want to undo the last commit you can call:'
+                echo '    $ git reset --hard HEAD~1'
+                echo '* to roll back to the remote state:'
+                echo '    $ git reset --hard origin/[branch-name]'
+                echo
+                
+                set -x
+                version_file_path="./version/version.go"
+                
+                git fetch
+                git checkout ${RELEASE_START_BRANCH}
+                
+                current_version="$(go run _scripts/get_version.go -file "$version_file_path")"
+                bumped_version=$(ruby -e "splits='${current_version}'.split('.');major=splits[0];minor=splits[1];patch=splits[2];puts \"#{major}.#{minor}.#{patch.to_i.next}\"")
+                
+                bash _scripts/set_version.sh "$version_file_path" "$bumped_version"
+                
+                git add "$version_file_path"
+                git commit -m "v${bumped_version} [skip ci]"
+                echo
+                echo '------------'
+                git show
+                echo '------------'
+                echo
+                
+                git checkout ${RELEASE_DESTINATION_BRANCH}
+                git pull
+                git merge ${RELEASE_START_BRANCH} --no-ff -m "Merge ${RELEASE_START_BRANCH} into ${RELEASE_DESTINATION_BRANCH}, release: v${bumped_version}"
+                git tag "${bumped_version}"
+                echo
+                echo '------------'
+                git show
+                echo '------------'
+                echo
+                
+                git checkout ${RELEASE_START_BRANCH}
+  configure-upstream:
```

Comment:
Pls create a workflow that contains all the necessary workflows needed for the version bump.